source/query-with-natural-language/prompt-natural-language-aggregation.txt

```diff
@@ -0,0 +1,167 @@
+.. _prompt-natural-language-agg:
+
+==============================================
+Prompt your First Natural Language Aggregation
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+You can use MongoDB Compass to generate aggregation queries using natural 
+language. Compass uses AI to generate aggregations based on prompts 
+you provide. Querying with natural language can be a helpful starting 
+point and assist you in learning to write MongoDB queries.
+
+About this Task
+---------------
+
+.. note::
+
+   .. include:: /includes/fact-natural-language-query.rst
+
+- You can also provide natural language prompts on the 
+  :ref:`documents tab <prompt-natural-language-query>`.
+
+- This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+  collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable natural language querying in Compass<enable-natural-language-querying>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Navigate to the :guilabel:`Natural Language Query Bar`
+      
+      I. Select the :guilabel:`Aggregations` tab.
+      #. Click the :guilabel:`Generate Aggregation` button.
+
+      The :guilabel:`Natural Language Query Bar` displays.
+
+   .. step:: Type a question about your collection
+
+      Type a natural language prompt for your collection into the query 
+      bar. Aggregation pipeline prompts usually have an aggregation verb 
+      such as count, average, or sum with logical conditions. For 
+      example: ``How many movies have more than 3 writers 
+      in the writers array?``
```

Comment:
Do we really want to mention particular stages here? Since a user is already on the aggregation tab (like we do not redirect them) they can use any stage like $match for example.